### PR TITLE
Simplified para-registrar-swap

### PR DESCRIPTION
**Pull request by jboetticher**
Replaced sections with the wrappers

### CODE REVIEW

**Comment by albertov19** [comment]
Hey @jboetticher this is still not following the sort of guidelines from the other scripts. Check https://github.com/PureStake/xcm-tools/blob/main/scripts/generic-call-proposer.ts#L47-L95 as an example

File: scripts/para-registrar-swapper.ts [hunk 1]
```diff
@@ -1,118 +1,120 @@
...
+        const collectiveThreshold = args['collective-threshold'] ?? 1;
+
+        await democracyWrapper(
+            api,
+            args["send-proposal-as"],
+            preimage,
+            proposalAmount,
+            account,
+            nonce,
+            collectiveThreshold
```
Comment (albertov19) [inline scripts/para-registrar-swapper.ts:116]: We need to add support for Gov V2 here right?

**Review by albertov19 — CHANGES_REQUESTED** [review]
Missing Gov V2 support 😬

**Comment by jboetticher** [comment]
Well, I rebased to get OpenGovV2 changes into this branch. Not sure if that was the right git action, but it is what it is. Main changes are README & para-registrar-swapper.ts.

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**